tests/e2e/utils/utils.py

```diff
@@ -367,27 +370,52 @@ def exec_shell(cmd):
     if completedProcess.returncode != 0:
         raise Exception(f"ERROR: Failed to execute shell command \n{cmd}")
 
+
 def get_variable_from_params(path, var_name):
     with open(path) as f:
         for line in f:
             if var_name in line:
                 return line.split("=")[1].strip()
 
+
 def find_and_replace_in_file(path, old_val, new_val):
-    with open(path, 'r') as file :
+    with open(path, "r") as file:
         filedata = file.read()
     filedata = filedata.replace(old_val, new_val)
-    with open(path, 'w') as file:
+    with open(path, "w") as file:
         file.write(filedata)
 
+
 def check_helm_chart_exists(chart_name, namespace):
     if namespace:
-        retcode = subprocess.call(
-            f"helm status {chart_name} -n {namespace}".split()
-        )
+        retcode = subprocess.call(f"helm status {chart_name} -n {namespace}".split())
     else:
         retcode = subprocess.call(f"helm status {chart_name}".split())
 
     if retcode == 0:
         return True
-    return False
\ No newline at end of file
+    return False
+
+
+def create_addon(addon_name, cluster_name, account_id, role_name):
```

Comment:
Can you add a region parameter for create_addon and delete_addon, will be useful when running integ tests.

Reply:
added